src/components/BookItem.js

```diff
@@ -17,10 +17,13 @@ class BookItem extends React.Component {
 }
 
 BookItem.propTypes = {
-  book: PropTypes.string.isRequired,
-  title: PropTypes.string.isRequired,
-  author: PropTypes.string.isRequired,
-  deleteBooksProps: PropTypes.func.isRequired,
+  // eslint-disable-next-line react/require-default-props
+  book: PropTypes.element({
+    title: PropTypes.string,
+    author: PropTypes.string,
+  }),
+  // eslint-disable-next-line react/require-default-props
+  deleteBooksProps: PropTypes.func,
 
 };
```

Comment:
- [ ] The app is not running because of a prototype error, I recommend fixing this error so your project is complete:

<img width="251" alt="image" src="https://user-images.githubusercontent.com/77190203/166456022-a2f3b172-b260-47d8-9b5c-5dc4fcbc9af0.png">
 
- Please make sure that your app runs successfully before requesting another review 😃.

Reply:
Thank you for the review. I restated the propTypes for each prop and it works properly now.